### PR TITLE
MORE FEET

### PR DESCRIPTION
**Pull request by lipamanka**
you asked for more feet so I am sending you more feet but now in the form of high resolution PNGs!

### CODE REVIEW

**Comment by lipamanka** [comment]
I added new cat SVG and PNG files

**Review by saizai — DISMISSED** [review]
Please
1. rename UNLWSfoo.png to just foo.png (etc), so it's the same file name as the svg other than the svg vs png
2. rename "kneel" to "keel" — that's for eg the bottom of a boat, floating log, water fly, snowshoes, trampolines, etc where the surface deforms

File: files/UNLWSanotomicalfoot.png
Comment (saizai) [inline files/UNLWSanotomicalfoot.png:1]: should be `anatomical_foot.png` to match

File: files/UNLWSmovingfoot.png
Comment (saizai) [inline files/UNLWSmovingfoot.png:1]: should be `moving_foot.png` to match

**Comment by saizai** [comment]
would probably look better if each of the swoops is identical — same baseline (bottom aligned) and curve, except
- for 'keel' they binds to the foot hook rather than the ground line, because the ground line fuses / deforms to the foot hook
- for 'root' the first one is a continuation of the foot hook, plus two swoops — its entry angle may have to be different than the swoops to use the same hook shape for all, but past the bottom-most point its swoop part should probably be identical

this is just aesthetic, not grammatical, so it's subject to override if it would look better a different way

**Comment by saizai** [comment]
Also probably indicate the G BP — it's on the left side of the ground line for all the foot-oids. Ground line is straight and parallel for all but keel (for which think of it like the dimple in water surface tension of a water fly's foot).

see eg
- middle & top UB27 UB27
- bottom right UB36 UB36
-  left & bottom UB37 UB37

File: files/kneel.svg
Comment (saizai) [inline files/kneel.svg:1]: rename `keel.svg` - check if diff vs current one

**Comment by lipamanka** [comment]
when it comes to file names, you can change them! But I'll change these.

**Comment by saizai** [comment]
> when it comes to file names, you can change them! But I'll change these.

I can, and did, but then yours create new files when you try to merge again. like https://github.com/saizai/unlws/pull/5/files#diff-a1e36bda0628755161f0222c6cb26981bd6a09a2a72f561fc703bb40cee8edb7

**Comment by saizai** [comment]
@lipamanka  Looks improved. /files/foo.png & /files/foo.svg, or /files/png/foo.png & /files/svg/foo.svg, are both fine methods — but pick one please. ;)

**Comment by lipamanka** [comment]
I am currently deleting things! I am NOT particularly experienced at github.

**Comment by saizai** [comment]
FYI in future you can just edit the file name to do a rename ;)

**Comment by lipamanka** [comment]
I also updated all the files to include the G BP though. otherwise I would have done that

**Review by saizai — APPROVED** [review]

**Comment by saizai** [comment]
merged and added to documentation as insert image by URL. thanks!

(see eg https://s.ai/nlws/files/png/keel.png etc)

**Pull request merged**